mypy/newsemanal/semanal.py

```diff
@@ -405,9 +405,11 @@ def add_implicit_module_attrs(self, file_node: MypyFile) -> None:
                 var = Var(name, an_type)
                 var._fullname = self.qualified_name(name)
                 var.is_ready = True
-                self.add_symbol(name, var, None)
+                self.add_symbol(name, var, dummy_context())
```

Comment:
The idea is to use `dummy_context()` instead of `None` to avoid some special cases.